### ISSUE TITLE
Error when opening ACNH

### ISSUE THREAD

**Issue by animeshadowpanda** [Team-Neptune/DeepSea#85]
Hi. Whenever I try to open ACNH on either sysNAND CFW or emummc, I get the 2155-8007 error. I've tried almost everything, and nothing works.
![20211118_191636](https://user-images.githubusercontent.com/94390422/142517168-508e4fbb-9b38-41dd-ad35-559b938dbfc5.jpg)

**Comment by hax4dazy** [comment]
Are you on the latest DeepSea?

**Comment by animeshadowpanda** [comment]
Yes, I am

**Comment by animeshadowpanda** [comment]
Yes, I am on the latest firmware. Also, the game was bought from the eShop before I hacked my switch, and it worked fine.

**Comment by Slluxx** [comment]
If this issue is not resolved, please request help at our discord server.
